.github/CODEOWNERS

```diff
@@ -32,3 +32,9 @@ Cargo.toml @didier-wenzek @albinsuresh @jarhodes314 @rina23q
 
 # Python
 *.py @reubenmiller @albinsuresh @jarhodes314
+
+# Robot Framework
+*.robot @gligorisaev
+
+# Markdown
+*.md @gligorisaev
```

Comment:
It would make more sense to add `@gligorisaev` to the `/docs/` rule instead of adding a new .md rule

Reply:
Changed